### ISSUE TITLE
beep does not change with NVDA output device changes

### ISSUE THREAD

**Issue by cary-rowen** [mltony/nvda-word-nav#5]
Press NVDA+Ctrl+S to change the output device from headphones to speakers, then confirm.
The sound of NVDA is already playing in the speakers, and the beep of Word-Nav is still in the headphones.
Expected performance should be consistent with NVDA's audio output
Also, Sentence-Nav works as expected

**Comment by cary-rowen** [comment]
Hi @mltony 
Beeper's implementation looked complicated, but I was able to fix this using the processing in Sentence-nav.
Would you like me to fix it? Or you have other ideas.

Thanks
Cary
